apps/backoffice-v2/src/common/components/organisms/Header/hooks/useNavbarLogic/useNavbarLogic.tsx

```diff
@@ -57,9 +63,13 @@ export const useNavbarLogic = () => {
   const { pathname } = useLocation();
   const checkIsActiveFilterGroup = useCallback(
     (navItem: TRouteWithChildren) => {
-      return navItem.children?.some(
-        childNavItem => childNavItem.filterId === filterId || childNavItem.href === pathname,
-      );
+      if (navItem.children) {
+        return navItem.children?.some(
+          childNavItem => childNavItem.filterId === filterId || childNavItem.href === pathname,
+        );
+      } else {
+        return navItem.href === pathname;
+      }
```

Comment:
Let's revert this considering the other suggested changes. 🙂  Also consider the following in the future for readability by early returning on the simpler condition (usually the negative).
```suggestion
     if (!navItem.children) {
        return navItem.href === pathname;
     }

     return navItem.children?.some(
       childNavItem => childNavItem.filterId === filterId || childNavItem.href === pathname,
     );
```